README.de_DE.md

```diff
@@ -0,0 +1,83 @@
+<p align="right">🌎 <a href="https://github.com/redaxo/redaxo/blob/master/README.md">English</a></p>
+
+<p align="center"><a href="https://redaxo.org"><img src="https://raw.githubusercontent.com/redaxo/redaxo/assets/redaxo-logo.svg" alt="REDAXO" width="280px"></a></p>
+
+<h3 align="center">PHP-CMS seit 2004. Einfach, flexibel, sinnvoll.</h3>
```

Comment:
PHP-CMS -> REDAXO CMS

Reply:
Eigentlich sollte das beschreiben, worum es geht, nämlich Content-Management auf Basis von PHP. Bei Twitter, wo wenig Platz ist, hatte ich damals schlicht »PHP CMS« draus gemacht und das hier für die README übernommen.

»REDAXO CMS« an dieser Stelle fände ich unpassend, wie wäre es denn mit »Content-Management seit 2004« und »Content Management since 2004«?

Reply:
“a PHP-based CMS since 2004“?

Reply:
Ich bin mit euren Vorschlägen alle einverstanden.

Hier vllt nochmal was mich daran konkret „gestört“ hat: Der Name des CMS kommt nicht in der headline vor

Reply:
`<h1>` taugt nicht, weil die Styles einen border enthalten. Dann also doch `<p>`.
Ich habe mal das Wording auf »Content Management« geändert. Was meint ihr? Oder doch in Richtung von Alex’ Vorschlag »PHP-based CMS since 2004«? Mir alles recht.